ERCS/erc-7717.md

```diff
@@ -0,0 +1,250 @@
+---
+eip: 7717
+title: Deterministic AA wallet
+description: Propose a non-custodial interface for service providers to provide AA accounts.
+author: Jinseok Park (@mellowcroc), Jaehun Kim (@jaehunkim), DrKeccak (@drkeccak)
+discussions-to: https://ethereum-magicians.org/t/eip-7710-deterministic-aa-wallet/20080
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-05-22
+requires: 4337
+---
+
+## Abstract
+
+Propose a non-custodial interface for service providers to provide AA accounts.
```

Comment:
This is just your description again. Your abstract should contain enough detail to give the reader a high-level (but still technical) overview of how your proposal accomplishes its goal.